locales/en-US/default.json

```diff
@@ -606,7 +616,7 @@
     "offline": "Offline",
     "online": "Online",
     "opals": "Opals",
-    "overallWins": "Overall Wins",
+    "overallWins": "Total Wins",
```

Comment:
Revert or make a new key for total wins. It should be overall because we use Solo Wins and Doubles Wins. So overall stats should follow the pattern of mode name followed by wins.